doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -712,10 +714,13 @@ Libraries for networking
 
     * Support for using pre-provisioned X.509 certificates.
     * Support for using DTLS Connection Identifier
+    * Support for MCUmgr SMP client for external FOTA support with shell.
```

Comment:
```suggestion
    * Support for MCUmgr SMP client when using a shell command to perform a FOTA on an external SoC.
```
This need your careful attention. I could not really make heads or tails of this, but I tried.

Reply:
Support for MCUmgr SMP client to perform a FOTA on an external SoC

Reply:
Sounds a lot better.